exampleservice/CustomerService/Events/InvalidSessionEvent.cs

```diff
@@ -1,9 +1,9 @@
 using System;
 using exampleservice.Framework.BaseFramework;
 
-namespace exampleservice.CustomerService.Contract
+namespace exampleservice.CustomerService.Events
 {
-    public class SessionTimeoutEvent : EventBase
+    public class InvalidSessionEvent : EventBase
```

Comment:
Shouldn't we differentiate between timeout (expected error) and a general error (like an invalid session-id)?

Reply:
the reason for the rename was the multiple usage of session timout even for non-session timeout errors. so we pursure to only use one positive an one negativ event.
for a real project, this wouldn´t be a good idea and the whole error handling should be treated more clearly

Reply:
fine for me. With the new structure it makes even more sense now :+1: